python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py

```diff
@@ -106,6 +111,10 @@ def for_run_failure(self):
             instance=self.instance,
         )
 
+    @property
+    def pipeline_run(self) -> PipelineRun:
```

Comment:
maybe throw a deprecation warning?

Reply:
Is it weird to say `as of 0.13.0`?  We'd already be at `0.13.10`